tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_skip_tour.py

```diff
@@ -40,19 +40,27 @@ def test_ftu_skip_tour(self):
         ssid = self.testvars['wifi']['ssid']
         psk = self.testvars['wifi'].get('psk')
         keymanagement = self.testvars['wifi'].get('keyManagement')
+        share_data_default = True
```

Comment:
Yep, I've seen that post. Think there are a few differences:

* The variable they're creating is a partial "base" value with no real semantic importance, then they're computing in the test with concatenation. That causes the "two slashes" issue.
* The function is so small that there's minimal value in renaming the literal just to give it a name.
* The URL itself is self-documenting.
* The Google URL isn't likely to change.

OTOH, if at the top of file or in a commons file or whatever they'd had:

* google_plus_url = 'http://plus.google.com/'
* google_plus_photos_url = 'http://plus.google.com/u/0/photos'

...then I would think using those values in the test would be better. Note that there's no concatenation--string building of constants is usually bad for other reasons, especially localization. And that's one reason this is arguably better, those strings can be localized. 

But as far as what I did:

* The value stands alone with semantic meaning: the default is indeed True, period.
* The only computation is "not", which doesn't have the same issues as concatenation and is clear by itself.
* It's a large, sprawling test scenario function. That has its own possible issues, but I'm working in existing code.
* The sharing default is likely to change.

As for your added code comments, this is the most important part:

When I test the value, I really do mean to test that the value matches the system default, not that the value specifically matches True. 

That's very, very important in terms of communicating code meaning. Assigning a name means you're assigning a level of abstraction. 

And further, writing this from scratch, that default value would come from a data file of system defaults. They're external givens and not otherwise important to the intention of the test.

I suspect that's the difference here. You think the test should test for True, but I disagree. It should test that the initial UI value matches the default system value. 

And honestly, I disagree with that too, which is why I didn't hardcode the test the first time--the actual functionality of the UI is to come in with the *current* system value, not the default system value. As it stands, the test isn't idempotent without a reset, because the test itself changes that value so when it's rerun it will fail.

So I guess, if anything, what I would say is this test is less robust because we've mixed back-end testing concerns with UI testing concerns. But since that was at your request, I think we both need to roll with that one.